types/twitch/twitch-tests.ts

```diff
@@ -0,0 +1,123 @@
+const players: Twitch.Player[] = [
+    new Twitch.Player('id', {
+        width: 100,
+        height: 100,
+        parent: ['yourUrl.com']
+    }),
+    new Twitch.Player("id", {
+        width: "100%",
+        height: "100%",
+        video: "videoId",
+        parent: ['yourUrl.com'],
+    }),
+    // @ts-expect-error
+    new Twitch.Player("id", {
+        width: "100%",
+        height: "100%",
+    }),
+    new Twitch.Player("id", {
+        width: "100%",
+        height: "100%",
+        video: "videoId",
+        parent: ['yourUrl.com'],
+        autoplay: true,
+        muted: true,
+        time: "1h1m1s"
+    }),
+    new Twitch.Player("id", {
+        width: "100%",
+        height: "100%",
+        collection: "videoId",
+        parent: ['yourUrl.com'],
+        autoplay: true,
+        muted: true,
+        time: "1h1m1s"
+    }),
+    new Twitch.Player("id", {
+        width: "100%",
+        height: "100%",
+        channel: "videoId",
+        parent: ['yourUrl.com'],
+        autoplay: true,
+        muted: true,
+        time: "1h1m1s"
+    }),
+];
+
+const player = new Twitch.Player("id", {
+    width: "100%",
+    height: "100%",
+    channel: "videoId",
+    parent: ['yourUrl.com'],
+    autoplay: true,
+    muted: true,
+    time: "1h1m1s"
+})
+// $ExpectType void
+player.addEventListener(Twitch.Events.READY, () => { })
```

Comment:
are you sure about the names for literals here? The docs seem to list different ones:
https://dev.twitch.tv/docs/embed/video-and-clips/#javascript-events

Reply:
I just reworked it. The docs just name the location of the strings in runtime but not their values, and i typed them by value by getting them from the code. But I'll just typed them by location now as "string". it's probably better in case the actual strings change in an update.

thanks for the review